doc.go

```diff
@@ -195,6 +195,10 @@ So it's just fine to check an error like this:
 
     c.Assert(err, qt.IsNil)
 
+IsNotNil
+
+IsNotNil returns a Checker negating the IsNil checker. IsNotNil is the equivalent of qt.Not(qt.IsNil)
```

Comment:
Please add the usage example here as well.
And I think we should regenerate the readme, but I can do this later.
Related issue: https://github.com/frankban/quicktest/issues/98

Reply:
ok, got it - give me a bit to refactor. I'll try and get to it today, tomorrow at latest. Thanks!

Reply:
Completed in latest commit.